CHANGES.md

```diff
@@ -8,6 +8,7 @@ Change Log
 * Fixed a crash when `viewer.zoomTo` or `viewer.flyTo` were called immediately before or during a scene morph [#2775](https://github.com/AnalyticalGraphicsInc/cesium/issues/2775).
 * Fixed an issue where `Camera` functions would throw an exception if used from within a `Scene.morphComplete` callback [#2776](https://github.com/AnalyticalGraphicsInc/cesium/issues/2776).
 * `Model` can now load Binary glTF from a Uint8Array.
+* Added `UrlTemplateImageryProvider`.
```

Comment:
I updated `CHANGES.md` to better explain how it can be used.  And I changed the examples so that they all actually work.  I think the CartoDB Positron (or Dark Matter) map would be a nice one to show in the release blog.  It was possible to access it with the OSM provider before, but with the new one we get the multi-host performance boost.